solr/distribution/build.gradle

```diff
@@ -50,40 +50,30 @@ if (project.ext.useGpgForSigning) {
 // Prepare the "source" distribution artifact.
 apply from: buildscript.sourceFile.toPath().resolveSibling("source-release.gradle")
 
-// Prepare the "binary" distribution artifact.
-apply from: buildscript.sourceFile.toPath().resolveSibling("binary-release.gradle")
-
-// Configure artifact signing.
-apply from: buildscript.sourceFile.toPath().resolveSibling("artifact-signing.gradle")
-
 
 // Set up the HTML-rendered "changes" distribution artifact by linking to documentation's output.
 configurations {
   changesHtml
-  solrTgz
-  solrTgzSignature
   docker
 }
 
-artifacts {
-  solrTgz assembleBinaryTgz
-  solrTgzSignature file(assembleBinaryTgz.archiveFileName.get() + ".asc")
-}
-
 dependencies {
   changesHtml project(path: ":solr:documentation", configuration: "changesHtml")
   docker project(path: ':solr:docker', configuration: project.ext.withSignedArtifacts ? 'packagingOfficial' : 'packaging')
 }
 
+def distTarTask = rootProject.getTasksByName("distTar", true)[0]
+def distZipTask = rootProject.getTasksByName("distZip", true)[0]
```

Comment:
Is there a more consise way to reference this task than this verbose one?

Reply:
I tried a few things and nothing worked. This can be fixed later if possible.